IBPSA/BoundaryConditions/Validation/UsersGuide.mo

```diff
@@ -0,0 +1,163 @@
+<li>Average relative humidity (%)</li>
+<li>Average dewpoint temperature (&deg;C)</li>
+<li>Average humidity ratio (kg moisture/kg dry air)</li>
+<li>Average wet bulb temperature (&deg;C)</li>
+<li>Sum of total, beam, and diffuse solar radiation incident on each surface (Wh/m2)</li>
+</ul>
+<br><p><i>Hourly Outputs</i></p>
+<p>The following outputs are  provided for each hour of the days specified for each test case in Table 3:</p>
+<ul>
+<li>Dry bulb temperature (&deg;C)</li>
+<li>Relative humidity (%)</li>
+<li>Dewpoint temperature (&deg;C)</li>
+<li>Humidity ratio (kg moisture/kg dry air)</li>
+<li>Wet bulb temperature (&deg;C)</li>
+<li>Windspeed (m/s)</li>
+<li>Wind direction (degrees from north)</li>
+<li>Station pressure (mbar)</li>
+<li>Total cloud cover (tenths of sky)</li>
+<li>Opaque cloud cover (tenths of sky)</li>
+<li>Sky temperature (&deg;C)</li>
+<li>Sum of total, beam, and diffuse solar radiation incident on each surface (Wh/m2)&nbsp;</li>
+</ul>
+<br><p><i>Table 3: Specific Days for Output</i></p>
+<table summary = \"Specific Days for Output\" cellspacing=\"2\" cellpadding=\"0\" border=\"1\"><tr>
+<td><p>Case </p></td>
+<td><p>Days</p></td>
+</tr>
+<tr>
+<td><p>WD100 </p></td>
+<td><p>May 4th, July 14th, September 6th</p></td>
+</tr>
+<tr>
+<td><p>WD200 </p></td>
+<td><p>May 24th, August 26th</p></td>
+</tr>
+<tr>
+<td><p>WD300 </p></td>
+<td><p>February 7th, August 13th</p></td>
+</tr>
+<tr>
+<td><p>WD400 </p></td>
+<td><p>January 24th, July 1st</p></td>
+</tr>
+<tr>
+<td><p>WD500 </p></td>
+<td><p>March 1st, September 14th</p></td>
+</tr>
+<tr>
+<td><p>WD600 </p></td>
+<td><p>May 4th, July 14th, September 6th</p></td>
+</tr>
+</table>
+<br><p><i>Sub-hourly Outputs</i></p>
+<p>The following outputs are  provided at each timestep of the days specified for each test case in Table 3:</p>
+<ul>
+<li>Dry bulb temperature (C)</li>
+<li>Relative humidity (%)</li>
+<li>Sum of total, beam, and diffuse solar radiation incident on each surface (Wh/m2) </li>
+</ul>
+<p>The following outputs are  provided integrated hourly for the days specified for each test case in Table 3:</p>
+<ul>
+<li>Total incident horizontal solar radiation (Wh/m2)</li>
+<li>Total incident horizontal beam solar radiation (Wh/m2)</li>
+<li>Total incident horizontal diffuse solar radiation (Wh/m2) </li>
+</ul>
+<h4>Validation results</h4>
+<p>(Not available yet)</p>
+<h4>Implementation</h4>
+<p>To generate the data shown in this user guide, run </p>
+<pre>
+cd IBPSA/Resources/Data/BoundaryConditions/Validation/BESTEST
+python3 generateResults.py -p
+</pre>
+<p>At the beginning of the Python script there are several options that the user can choose, by default the script will:
+</p>
+<ul>
+<li>Clone the last master branch of the IBPSA repository into a temporary directory</li>
+<li>Execute all the simulations and create the folders with the .mat and .json files inside the BESTEST/Simulations folder</li>
+</ul>
+<h4>References</h4>
+<p>(Not available yet)</p>
```

Comment:
Why are the references not available yet?

Reply:
All the teams partecipating in the BESTEST had one issue or the other. The idea is to have the results ready to the ASHRAE meeting in Jan 2021